cd2h_repo_project/modules/frontpage/templates/cd2hrepo_frontpage/index.html

```diff
@@ -30,16 +30,32 @@
     <p>
-      Research Data Management (RDM) platforms play a critical role in the research ecosystem to preserve and share, enable reproducibility, and empower reuse of collaborative research activities and outputs.
+      Research Data Management (RDM) platforms make it possible to preserve and share, support reproducibility, and empower reuse of collaborative research activities and outputs.
     </p>
 
     <p>
-      <b>InvenioRDM</b> makes it possible for organizations to securely house research products and make them <b>discoverable, shareable, and citable</b> -- from publications and presentations to datasets, software, lay summaries, policy documents, and workflows. InvenioRDM is being developed as part of a large, multi-organization collaboration in partnership with the <b><a href="https://home.cern/">European Organization for Nuclear Research (CERN)</a></b>, developers of <b><a href="https://zenodo.org">Zenodo</a></b> for the European community.  Together, we are working to build a wide range of features that can help <b>power biomedical research</b> and support data sharing, accessibility, innovation, and interdisciplinary collaboration.
+      <b>InvenioRDM</b> enables organizations to securely house research products and <b>make them discoverable, shareable, and citable</b> -- from publications and presentations to datasets, software, lay summaries, policy documents, and guidelines. <b>InvenioRDM</b> is being developed as part of a large, multi-organization collaboration in partnership with the <a href="https://home.cern/">European Organization for Nuclear Research (CERN)</a>, birthplace of the World Wide Web and developers of the <a href="https://zenodo.org/">Zenodo</a> RDM for the European community. Together we are working to build a wide range of features that can help power biomedical research and support data sharing, innovation, knowledge dissemination, and interdisciplinary collaboration.
     </p>
 
+    <p>
+      Our work is driven by user needs and informed by best practices and standards, including those that help define <a href="https://ngr.coar-repositories.org/">Next Generation Repositories</a> as a foundation for a distributed, globally-networked infrastructure for scholarly communication, discovery, and innovation.
+    </p>
+
+    <br>
+
+    <div class="call-to-action">
+      <p>
+        <b>Contact us</b>. <a href="{{ url_for('contact_us.contact_us') }}">Email us</a> to ask questions or if you would like a test username and password to create records of your own.
+      </p>
+      <p>
+        <b>Join us</b>. Get updates and test new releases on the <a href="https://invenio-software.org/">product site</a>, <a href="{{ url_for('contact_us.contact_us') }}">email us</a> to request a demonstration or chat, take a look at our competitive landscape analysis, and get involved with collaboration and discussion through our <a href="https://github.com/inveniosoftware/invenio-app-rdm">open source project</a>.
```

Comment:
I updated this block with new links today (9/23/2019) in another PR. "competitive landscape analysis" now has a link and "open source project" now links to the CD2H InvenioRDM repo.

Reply:
Same, I will also wait to rebase.